docker/base-openjre/Makefile

```diff
@@ -2,9 +2,11 @@ OPENJRE_VERSION := "8u181-b13-2~deb9u1"
 OPENJRE_IMAGE_TAG := stretch-$(subst ~,-,$(OPENJRE_VERSION))
 OPENJRE_IMAGE := $(DOCKER_REPO)/base-openjre:$(OPENJRE_IMAGE_TAG)
 
+BUILD_COMMAND := $(if $(shell which buildah),buildah build-using-dockerfile,docker build)
```

Comment:
If we could just have build command default to `docker build` and leave the buildah components out of it. You should still be able to over ride it via BUILD_COMMAND

I think that is what @zegelin was getting at

Reply:
Yes exactly. `BUILD_COMMAND` wouldn't just be the name of the binary to run (`docker`, `build`, etc) but can also contain any arguments required (i.e `docker build`, `buildah bud`).

So I propose:
- A global include Makefile in the root directory that defines `BUILD_COMMAND` and sets it to `docker build`.
- For alternate build commands, just invoke `make BUILD_COMMAND='builda bud'`